packages/core/src/traits/identifiable.ts

```diff
@@ -0,0 +1,10 @@
+import { Trait } from "@ddd-ts/traits";
+import type { Identifier, IIdentifiable } from "../interfaces/identifiable";
+
+export const Identifiable = Trait((base) => {
```

Comment:
why Identifiable, but Named? Identified and Named or Identifiable or Nameable right?